docs/.vitepress/config.js

```diff
@@ -38,24 +38,26 @@ module.exports = {
 
     sidebar: [
       {
-        text: 'Getting Started',
-        link: '/guide/'
+        text: 'Introduction',
+        link: `/guide/`
       },
       {
-        text: 'Integration',
-        link: '/guide/integration'
-      },
-      {
-        text: 'Postgres Schema',
-        link: '/guide/schema/',
-        children: getMartenPGSchemaSidebar()
-      },
-      {
-        text: 'Commandline Tooling',
-        link: '/guide/cli'
+        text:'Integration and Configuration',
+        link: '/guide/configuration/',
+          children: [
+              {text: 'Bootstrap with HostBuilder', link: '/guide/configuration/hostbuilder'},
+              {text: 'Do It Yourself IoC Integration', link: '/guide/configuration/ioc'},
+              {text: 'Command Line Tooling', link: '/guide/configuration/cli'},
+              {text: 'Configuring Document Storage with StoreOptions', link: '/guide/configuration/storeoptions'},
+              {text: 'Json Serialization', link: '/guide/configuration/json'},
+              {text: 'Retry Policies', link:'/guide/configuration/retries'},
+              {text: 'Pre-Building Generated Types', link: '/guide/configuration/prebuilding'}
+
+          ]
       },
+
       {
-        text: 'Document DB',
+        text: 'Document Storage',
         link: '/guide/documents/',
         children: getDocumentDbSidebar()
```

Comment:
I think we can remove these methods and inline the objects, this will improve readability in terms of structure and also juggling things. I will do this change once we pull in this docs.